File: Array/largestPalindrom.py
def checkPrime(num):
    n = num
    rev = 0
    while n > 0:
        a = n % 10
        rev = rev*10+a
        n = n//10
    if num == rev:
        return True
    return False

def largestPrime(lst):
    mx = 0
    for i in lst:
        if checkPrime(i):
            c = len(str(i))
            if c >= mx:
                mx = c
                num = i
    print(f'largest prime {num} with {mx} max')
    print()

print("enter list element")
n = [1, 121, 12321, 11, 123456, 34567876543]
largestPrime(n)

File: Array/largestPalindrom.java
import java.util.Scanner;
import static java.lang.System.out;
class DeleteArr
{ 

 static boolean checkPalindrom(int num)
 {
  
    int n = num;
    int rev = 0;
    while(n>0)
    {
       int a = n % 10;
       rev = rev * 10+ a;
       n = n / 10; 
    }
    if(rev==num)
    return true;
    return false;
 }
 public static void main(String args[])
 {
  Scanner sc = new Scanner(System.in);
  out.println("Enter size of arr");
  int n = sc.nextInt();
  out.println("Enter array element");
  int arr[] = new int[n];
  for(int i=0;i<n;i++)
  {
   arr[i] =sc.nextInt();
  }
  int max = 0;
  for(int i=0;i<n;i++)
  { 
      int count = 0;
      if(checkPalindrom(arr[i]))
      {
              //implementation remaining
      }
  }
  sc.close();
 }
}